### PR TITLE
Update solver plugin docstring/example

### PR DESCRIPTION
**Pull request by beeankha**
The current solver plugin API example is missing the `logging` import and a few other lines. One thing to troubleshoot is the how the `"My verbose solver!"` statement does not seem to print out in the terminal.

### CODE REVIEW

**Comment by beeankha** [comment]
Hrmm, for some reason the new example doesn't appear to be showing up on the [solver API docs page](https://continuumio-conda-docs--12192.com.readthedocs.build/projects/conda/en/12192/dev-guide/plugins/solvers.html) ☹️

**Comment by jezdez** [comment]
@beeankha The build output on readthedocs says:

```
/home/docs/checkouts/readthedocs.org/user_builds/continuumio-conda/checkouts/12192/conda/plugins/hookspec.py:docstring of conda.plugins.hookspec.CondaSpecs.conda_solvers:7: WARNING: Error in "code-block" directive:
maximum 1 argument(s) allowed, 11 supplied.
```

I think this is just a matter of adding more whitespace?

File: conda/plugins/hookspec.py [hunk 1]
```diff
@@ -29,9 +29,12 @@ def conda_solvers(self) -> Iterable[CondaSolver]:
         Example:
 
         .. code-block:: python
-
```
Comment (jezdez) [inline conda/plugins/hookspec.py:32]: This should not be removed :)

File: conda/plugins/hookspec.py [hunk 2]
```diff
@@ -29,9 +29,12 @@ def conda_solvers(self) -> Iterable[CondaSolver]:
...
             from conda import plugins
             from conda.core import solve
+            import logging
```
Comment (jezdez) [inline conda/plugins/hookspec.py:34]:
```suggestion
            import logging
            
            from conda import plugins
            from conda.core import solve
```

File: conda/plugins/hookspec.py [hunk 3]
```diff
@@ -46,7 +49,6 @@ def conda_solvers():
                     name="verbose-classic",
                     backend=VerboseSolver,
                 )
-
```
Comment (jezdez) [inline conda/plugins/hookspec.py:49]: Let's keep that.

**Review by jezdez — CHANGES_REQUESTED** [review]

File: conda/plugins/hookspec.py (hunk 2 above)
Comment (beeankha) [reply conda/plugins/hookspec.py]: Thank you for this suggestion @jezdez ! Looks like the import statements getting re-arranged enabled all of the original whitespace to pass linter checks. 👍

**Comment by beeankha** [comment]
Everything is showing up properly on the docs page now:
Screen Shot 2022-12-21 at 8 31 28 AM

**Review by jezdez — APPROVED** [review]

**Pull request merged**